easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.20-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,110 @@
+and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS
+platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples."""
+github_account = 'ndaniel'
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAT', '3.5'),
+    ('Bowtie', '1.2.3'),
+    ('Bowtie2', '2.3.5.1'),
+    ('BWA', '0.7.17'),
+    ('Kent_tools', '20190326', '-linux.x86_64', True),
+    ('SRA-Toolkit', '2.10.4'),
+    ('STAR', '2.7.2b'),
+    ('picard', '2.21.6', '-Java-11', True),  # Optional
+    ('Velvet', '1.2.10', '-mt-kmer_191'),  # Optional
+    ('OpenPyXL', '2.6.4'),  # Optional
+    ('parallel', '20190922'),  # Optional
+    ('pigz', '2.4'),
+    ('zlib', '1.2.11'),
+]
+
+local_bbmap_makefile = 'makefile.%s' % {'Darwin': 'osx', 'Linux': 'linux'}[OS_TYPE]
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': [GITHUB_SOURCE],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['4c66dbd7be64aaf52ab1f02278f06d9a8082b7f3f30a8045a50b4899e22a0385'],
+        'start_dir': '%(namelower)s-%(version)s',
+    }),
+    # FusionCatcher requires a fork of seqtk with features not available upstream
+    ('seqtk', '1.2-r101c', {
+        'easyblock': 'MakeCp',
+        'source_urls': ['http://github.com/ndaniel/seqtk/archive'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['4a57fbe828eccaa4f0122c84bf9c4c5409a024ad9deace31ec7bee6d25cf5e49'],
+        'start_dir': '%(name)s-%(version)s',
+        'files_to_copy': [(['seqtk'], 'tools/%(name)s')],
+    }),
+    # FusionCatcher requires this very specific version of BBMap
+    ('BBMap', '38.44', {
+        'easyblock': 'MakeCp',
+        'source_urls': [SOURCEFORGE_SOURCE],
+        'sources': ['%(name)s_%(version)s.tar.gz'],
+        'checksums': ['63ecdc83cdb0aaf5106b919df487f50a6b65e816d990501e9f74704c04b2a84a'],
+        'start_dir': '%(namelower)s',
+        'prebuildopts': "cd jni && ",
+        'buildopts': "-f %s" % local_bbmap_makefile,
+        'files_to_copy': [(['*'], 'tools/%(namelower)s')],
+    }),
+]
+
+# Set paths in the configuration file
+postinstallcmds = [
+    'sed -i "s|python = /usr/bin/|python = $EBROOTPYTHON/bin/|g" %(installdir)s/etc/configuration.cfg',
+    'sed -i "s|java = /usr/bin/|python = $EBROOTJAVA/bin/|g" %(installdir)s/etc/configuration.cfg',
+    'sed -i "s|paralell/src|parallel/bin|g" %(installdir)s/etc/configuration.cfg',
+]
+
+# Create symlinks in installation dir to dependencies from EB
+postinstallcmds += [
+    'ln -s $EBROOT%s %%(installdir)s/tools/%s' % (d, d.lower()) for d in [
+        'BLAT', 'BOWTIE', 'BOWTIE2', 'BWA', 'PARALLEL', 'PICARD', 'PIGZ', 'STAR', 'VELVET'
```

Comment:
BWA and paralell not needed

Reply:
BLAT is in Kent_tools. Should be probably used from there so dependency can be dropped.

Reply:
BWA and parallel are indeed used in FusionCatcher, please see my previous comment.
I'll check if we can drop BLAT in favour of Kent_tools.